src/main/java/com/sweetwith/dailynote/domain/reply/Reply.java

```diff
@@ -0,0 +1,47 @@
+package com.sweetwith.dailynote.domain.reply;
+
+import com.sweetwith.dailynote.domain.BaseTimeEntity;
+import com.sweetwith.dailynote.domain.posts.Post;
+import com.sweetwith.dailynote.domain.user.User;
+import com.sweetwith.dailynote.web.dto.ReplyRequestDto;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class Reply extends BaseTimeEntity {
+
+    @javax.persistence.Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    @Column(length = 500, nullable = false)
+    private String content;
+
+    @ManyToOne
+    @JoinColumn(name="Post_Id")
+    private Post post;
+
+    @ManyToOne
+    @JoinColumn(name="User_Id")
+    private User user;
+
+    @Column
+    private Long replyParentId;
+
+    @Column
```

Comment:
How about specify column name? If you don't specify column name, then column would be generated as you declared.
https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507